easybuild/easyconfigs/c/CNVnator/CNVnator-0.3-intel-2015a.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'MakeCp'
+
+name = 'CNVnator'
+version = '0.3'
+
+homepage = 'http://sv.gersteinlab.org'
+description = """Copy Number Variation discovery and genotyping from depth of read mapping."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+sources = ['%(name)s_v%(version)s.zip']
+source_urls = ['http://sv.gersteinlab.org/cnvnator/']
+
+dependencies = [
+    ('ROOT', '5.34.30'),
+]
+
+start_dir = 'src'
+
+prebuildopts = "sed -i -e 's|(ROOTSYS)/[a-z]*|&/root|g' Makefile && (cd samtools && make lib) &&"
```

Comment:
what is the sed magic doing here exactly?

is using SAMtools as a proper dependency an option, as opposed to building the SAMtools that is shipped with this?